python/ray/util/spark/start_ray_node.py

```diff
@@ -0,0 +1,77 @@
+import os.path
+import subprocess
+import sys
+import time
+import shutil
+import fcntl
+import signal
+
```

Comment:
Could you add some comments about this file: especially the process model. Seems start_ray_node.py is the sub-process and `ray start` is the sub-sub-process?